command-line.dylan

```diff
@@ -154,7 +154,8 @@ define function make-runner-from-command-line
                     progress: if (progress = $none) #f else progress end,
                     tags: parse-tags(get-option-value(parser, "tag")));
 
-  // TODO(cgay): runner-options are unused. Delete? What were they for?
+  // Options seem useful, but why are they positional rather than --option?
+  // i.e., what makes them so special?
```

Comment:
My thought process was basically that positional args are kind of an elevated position (pun intended) and "options" are rarely used so it would seem more natural to use the positional args to list the tests to run and remove `--test` and `--suite`. But either way works.